routing/router.go

```diff
@@ -390,6 +390,12 @@ func (r *ChannelRouter) Stop() error {
 	return nil
 }
 
+// GetSelfNode queries the current node identity public key used by the
+// ChannelRouter.
+func (r *ChannelRouter) GetSelfNode() route.Vertex {
```

Comment:
can just call it `SelfNode`. Getter is implied (unless it clashes with a member in ChannelRouter (which I dont think it does))